sktime/forecasting/compose/_fallback.py

```diff
@@ -112,6 +133,20 @@ def __init__(self, forecasters, verbose=False):
         self._anytagis_then_set("requires-fh-in-fit", True, False, self._forecasters)
         self._anytagis_then_set("capability:pred_int", False, True, self._forecasters)
 
+    def _validate_y_pred(self, y_pred):
+        if self.nan_predict_policy in ("warn", "raise"):
+            num_nans = y_pred.isna().sum()
```

Comment:
* should this use `isnull`? Or should it be an option to use `isna` vs `isnull`?
* `any` is faster than `sum` in this case